aip-console-jenkins/README.md

```diff
@@ -359,6 +360,7 @@ Under `Advanced Settings` you will find the following parameters :
     replaced by the current jobs' workspace path.
 * *Exclusion patterns* : List of file patterns to exclude in the delivery, separated by a comma. Each patterns needs to
   follow the syntax of [glob patterns](https://www.malikbrowne.com/blog/a-beginners-guide-glob-patterns)
+* *sleep duration* : Amount of seconds used to fetch the ongoing job status (defaulted to **1s**).
```

Comment:
I would replace "Amount" by "Number" because "amount" is related to a quantity of money